configs/katello/4.0.yaml

```diff
@@ -19,6 +21,8 @@
     :redmine_version_id: 1370
   :3.18.4:
     :redmine_version_id: 1396
+  :3.18.5:
```

Comment:
I think everything on 3.18.5 was specific to the KATELLO-3.18 branch ie it's not an actual prior - unless your cherry-picks actually contained something from 3.18.5?

Reply:
I think that everything indicated for picks were ignored because they were migration related.

Reply:
Ack. Can you remove 3.18.5 then?